src/Data/List/Relation/Binary/Subset/Setoid/Properties.agda

```diff
@@ -41,6 +42,18 @@ private
   variable
     a b p q r ℓ : Level
 
+------------------------------------------------------------------------
+-- Basics
+------------------------------------------------------------------------
+
+module _ (S : Setoid a ℓ) where
+
+  open Setoid S using (refl)
+  open Subset S
+
+  ⊈[] : ∀ {x xs} → x ∷ xs ⊈ []
+  ⊈[] p = Membershipₚ.∉[] S $ p (here refl)
+
```

Comment:
Suggest also adding the corollary to this?
```suggestion
  ⊆[]⇒≡[] : (_⊆ []) ⋐ (_≡ [])
  ⊆[]⇒≡[] {x = []}    _ = ≡.refl
  ⊆[]⇒≡[] {x = _ ∷ _} p with () ← ⊈[] p
```
for the cost of an additional initial
```agda
open import Relation.Binary.PropositionalEquality.Core as ≡ using (_≡_)
```